teamscale_precommit_client/precommit_client.py

```diff
@@ -90,14 +89,15 @@ def _format_findings(self, findings):
                     for finding in sorted_findings]
 
     def _print_findings(self, caption, findings, empty_line=False):
-        has_findings = self.log_to_stderr and len(findings) > 0
+        # Only log to stderr if there are findings
```

Comment:
Please describe why we shouldn't log to stderr when there's no findings. Will this display weirdly in QTCreator?